packages/web-components/src/components/cbp-tabs/cbp-tabs.stories.tsx

```diff
@@ -71,12 +79,14 @@ Tabs.args = {
       color: 'default',
       panelContent: 'Tab panel 1 content.',
       selected: false,
+      withIcon: true,
+      withBadge: true
     },
     {
       name: 'tab2',
       label: 'Tab 2',
       accessibilityText: '',
-      color: 'default',
+      color: 'danger',
```

Comment:
Can you change this back to default for the story? It's exposed for testing, but copying the default story code should not have test cases to fix/edit.

Reply:
updated